frontend/src/actionCreators/mineActionCreator.js

```diff
@@ -304,3 +304,30 @@ export const fetchMineBasicInfoList = (mine_guids) => (dispatch) => {
       dispatch(hideLoading());
     });
 };
+
+export const removeMineDocumentFromExpectedDocument = (mineDocumentGuid, expectedDocumentGuid) => (
+  dispatch
+) => {
+  dispatch(request(reducerTypes.REMOVE_MINE_EXPECTED_DOCUMENT));
+  dispatch(showLoading());
+  return axios
+    .delete(
+      ENVIRONMENT.apiUrl +
+        API.REMOVE_MINE_EXPECTED_DOCUMENT(expectedDocumentGuid, mineDocumentGuid),
+      createRequestHeader()
+    )
+    .then((response) => {
+      notification.success({
+        message: "Successfully removed the document from the report.",
+        duration: 10,
+      });
+      dispatch(success(reducerTypes.REMOVE_MINE_EXPECTED_DOCUMENT));
+      dispatch(hideLoading());
+      return response;
+    })
+    .catch(() => {
+      notification.error({ message: String.ERROR, duration: 10 });
```

Comment:
Does this endpoint return a custom error message? 
`String.ERROR` should be used as a backup.

Reply:
It does return a custom message I just missed this when I grabbed another action creator to base mine off of.